tests/Checks/AppKeyIsSetTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace BeyondCode\SelfDiagnosis\Tests\Checks;
+
+use Orchestra\Testbench\TestCase;
+use BeyondCode\SelfDiagnosis\Checks\AppKeyIsSet;
+
+/**
+ * @group checks
+ */
+class AppKeyIsSetTest extends TestCase
+{
+    /** @test */
+    public function it_checks_app_key_existance()
+    {
+        $check = app(AppKeyIsSet::class);
+        $this->assertFalse($check->check([]));
+
+        $this->app['config']->set('app.key', 'foo');
+
+        $this->assertTrue($check->check([]));
+    }
+
+    /**
+     * @test
+     */
+    public function it_returns_a_name_for_the_check()
+    {
+        $check = app(AppKeyIsSet::class);
+        $this->assertInternalType('string', $check->name([]));
+    }
```

Comment:
I wonder why you think tests of the `name` and `message` methods are necessary. The public interface of the `name()` method is `public function message(array $config): string`, which will throw a PHP error if you return something that is not a string. Yes, this will happen during runtime, but still... with a proper IDE, you should never be able to return something different. Or did I miss something?

Reply:
I want to check that the call succeed, another option is an assertTrue(true) after the check.

On this moment there isn't any complex name function. But with the message functions there where some functions that failed before if the check wasn't called. And because they where easy to add I found it a good idea to add them.